src/modules/private-share-folder/private-sharing.usecase.ts

```diff
@@ -38,6 +38,33 @@ export class PrivateSharingUseCase {
       roleUuid,
     );
   }
+
+  async updateRole(owner: User, privateFolderRoleId: string, roleId: string) {
+    const privateFolderRole =
+      await this.privateSharingRespository.findPrivateFolderRoleById(
+        privateFolderRoleId,
+      );
+
+    const folder = await this.folderRespository.findByUuid(
```

Comment:
Same as here: https://github.com/internxt/drive-server-wip/pull/170#discussion_r1250904534

Also, may be worth checking that the role in fact exists, before trying to update it, just in case something is wrong, we notify the user and we also are able to log the error as expected